src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -24,12 +24,12 @@ type BaseBreadcrumbsItem = {
     title?: string;
 };
 
-type LinkBreadcrumbsItem = {
+export type LinkBreadcrumbsItem = {
```

Comment:
Let's rename them to `BreadcrumbsLinkItem` and `BreadcrumbsButtonItem`, that's `Breadcrumbs` name will be at the beginning